garden_ai/globus_search/garden_search.py

```diff
@@ -3,26 +3,29 @@
 from garden_ai.gardens import Garden
 from globus_sdk import SearchClient, GlobusAPIError
 from pydantic import ValidationError
+from rich.traceback import install
 
 # garden-dev index
 GARDEN_INDEX_UUID = "58e4df29-4492-4e7d-9317-b27eba62a911"
 
+install()
+
 
 class RemoteGardenException(Exception):
     """Exception raised when a requested Garden cannot be found or published"""
 
+    pass
+
 
 def get_remote_garden_by_doi(doi: str, search_client: SearchClient) -> Garden:
     try:
         res = search_client.get_subject(GARDEN_INDEX_UUID, doi)
     except GlobusAPIError as e:
-        if e.code == 404:
-            raise RemoteGardenException(
-                f"Could not reach find garden with id {doi}"
-            ) from e
+        if e.http_status == 404:
+            raise RemoteGardenException(f"Could not find Garden with doi {doi}") from e
         else:
             raise RemoteGardenException(
-                f"Could not reach index {GARDEN_INDEX_UUID}"
+                f"Could not find Globus Search Index {GARDEN_INDEX_UUID}"
```

Comment:
nit: I don't think "find" is the right word here. It suggests that the user asked for the wrong search index, but the index could be right and we'd still hit this branch for a lot of different reasons. (eg, some transient network error)